run-tests-jenkins

```diff
@@ -2,3 +2,4 @@
 
 coverage run manage.py test -v 3 --with-xunit $1 --with-timer --exe
 coverage xml
```

Comment:
Can you coordinate with Dan and Martin on CUAHSI Jenkins, if this applies to that environment? Dan will be doing PyTest/Jenkins work.

Reply:
If Dan have any questions on this I will try to help.